_docs/concepts/handling-failures.md

```diff
@@ -0,0 +1,76 @@
+---
+category: Concepts
+title: Handling Failures
+
+parent: Traffic Management
+order: 30
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+Envoy sidecar/proxy provides a set of out-of-the-box _opt-in_ failure recovery features
+that can be taken advantage of by the services in an application. Features
+include:
+
+1. Timeouts
+2. Bounded retries with timeout budgets and variable jitter between retries
+3. Limits on number of concurrent connections and requests to upstream services
+4. Active health checks on each member of the load balancing pool
+5. Fine-grained circuit breakers -- applied per instance in the load
+   balancing pool
+
+These features can be dynamically configured at runtime through
+[Istio's traffic management rules](../tasks/timeouts-retries-circuit-breakers.html).
+
+The jitter between retries minimizes the impact of retries on an overloaded
+upstream service, while timeout budgets ensure that the calling service
+gets a response (success/failure) within a predictable timeframe.
+
+A combination of (periodic) active health checks and circuit breakers
+minimizes the chances of accessing an unhealthy instance in the load
+balancing pool. When combined with platform-level health checks (such as
+those supported in Kubernetes or Mesos), applications can ensure that
+unhealthy pods/containers/VMs can be quickly weeded out of the service
+mesh, minimizing the overall latency impact.
+
+Together, these features enable services in the Istio service mesh to fail
```

Comment:
Alternate ...

Together, these features enable the service mesh to tolerate failing nodes and to prevent localized failures from cascading instability to other nodes.